src/ZuluSCSI_disk.cpp

```diff
@@ -229,9 +229,22 @@ static void formatDriveInfoField(char *field, int fieldsize, bool align_right)
     }
 }
 
+// remove path and extension from filename 
+const char* get_image_name(const char* filename)
+{
+    char *j, *r;
+
+    r = (char *)malloc(strlen(filename));
+    strcpy(r, strrchr(filename,'/') + 5);
+    j = strrchr(r, '.');
```

Comment:
I think this will crash if the image name does not have an extension (no dot => strrchr returns null pointer).